packages/features/insights/components/RoutingFormResponsesTable.tsx

```diff
@@ -337,6 +347,26 @@ export function RoutingFormResponsesTableContent() {
       return [];
     }
     return [
+      columnHelper.accessor("formId", {
+        id: "formId",
+        header: t("routing_forms"),
+        enableColumnFilter: true,
+        enableSorting: false,
+        meta: {
+          filter: { type: "single_select" },
+        },
+        cell: () => null,
+      }),
+      columnHelper.accessor("bookingUserId", {
+        id: "bookingUserId",
+        header: t("people"),
+        enableColumnFilter: true,
+        enableSorting: false,
+        meta: {
+          filter: { type: "single_select" },
+        },
+        cell: () => null,
+      }),
```

Comment:
these are hidden columns but they exist to feed the filterable columns to the tanstack table.